common/scripted_triggers/wc_maa_related_triggers.txt

```diff
@@ -224,7 +224,16 @@ can_unlock_innovation_elephantry_trigger = {
 
 ### Has MaA tech (Used for buildings atm)
 
-has_unlocked_special_aerial_maa_trigger = {
+has_unlocked_aerial_maa_trigger = {
+	OR = {
+		has_innovation = innovation_taming_the_skies
+		has_innovation = innovation_flying_beasts
+	}
+	has_unlocked_special_aerial_maa_trigger = yes
+	has_unlocked_special_massive_aerial_maa_trigger = yes
```

Comment:
Everything should be inside `OR`. Right now it returns true if culture has both Taming the Skies and `has_unlocked_special_aerial_maa_trigger`.

`has_unlocked_special_aerial_maa_trigger` is renamed, no?